src/lib/libast/sfio/meson.build

```diff
@@ -22,3 +22,20 @@ libast_files += [
     'sfio/sfvprintf.c', 'sfio/sfvscanf.c', 'sfio/sfwalk.c', 'sfio/sfwr.c',
     'sfio/sfwrite.c'
 ]
+
+# number of bits in pointer
+charptr_size = cc.sizeof('char*')
```

Comment:
I'd prefer this be a void ptr but that's a nit and given the verbiage in the rest of the AST code it should probably remain framed as a "char ptr" for now. Which makes me a sad puppy :frown: